### ISSUE TITLE
What's good hyper parameter for finetuning on resnet

### ISSUE THREAD

**Issue by fairylrt** [facebook/fb.resnet.torch#7]
Maybe a stupid question.
What's the suggested weight decay and learning rate for finetuning?

**Comment by colesbury** [comment]
I have not experimented much with fine-tuning, but I would start with the default ResNet params:
LR = 0.1
weightDecay = 1e-4

And then decrease the learning rate by a factor of 10 when the validation error plateaus. The learning rate schedule is currently hard-coded into [train.lua](https://github.com/facebook/fb.resnet.torch/blob/master/train.lua#L159-L168), so you will probably have to change it there.
